packages/client/tests/functional/composite-index/list/prisma/_schema.ts

```diff
@@ -0,0 +1,26 @@
+import testMatrix from '../_matrix'
+
+export default testMatrix.setupSchema(({ provider }) => {
+  return /* Prisma */ `
+      generator client {
+        provider = "prisma-client-js"
+      }
+      
+      datasource db {
+        provider = "${provider}"
+        url      = env("DATABASE_URI_${provider}")
+      }
+      
+      type Location {
+        address    String
+      }
+      
+      model A {
+        id         String @id @map("_id")
+        name       String
+        locations   Location[]
+
+        @@unique([locations.address])
```

Comment:
What does it mean? Does it mean that:
* Two models cannot have the same list of addresses (via locations)
* Two models cannot have any address in common

Reply:
(not related to your PR really, but I'm curious to know)

Reply:
I don't know, raised something In slack https://prisma-company.slack.com/archives/C02FZENHD4N/p1660718192078639

Reply:
The first would imply the latter - So both. The slack thread solves this and coverage here: https://github.com/prisma/prisma/pull/14831/commits/657fa97df5642b75a2edd0e03cec80ff040c0847